addons/sys_server/fnc_masterIdTracker.sqf

```diff
@@ -137,9 +137,8 @@ if (GVAR(doFullSearch)) then {
                 };
             };
         };
+    } forEach ((ATLToASL [-1000,-1000,0]) nearObjects ["ACRE_baseRack", 1]);
```

Comment:
No, this works fine, look at the conversion. Otherwise we would end up not finding the objects. It was baffling me why it was not working.